### PR TITLE
fix language list for cantonese/yue

### LINKED ISSUE #1996: Change 'Chinese (Yue)' to 'Chinese (Cantonese/Yue)' in language list
Subject: Cantonese chinese not clear
Description:
nothing

Results:
Yue is not a commun word for Cantonese, I didn't understand, so better to mention Cantones?

Backend version: develop-b8a97954
Frontend version: develop-b8a97954
User Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36
Page: https://app.couchers.org/profile
User: amytje (4650)

### PR DESCRIPTION
**Pull request by ghost**
Closes [#1996](https://github.com/Couchers-org/couchers/issues/1996)

**Backend checklist**
- [x] Formatted my code by running `autoflake -r -i --remove-all-unused-imports src && isort . && black .` in `app/backend`
- [ ] Added tests for any new code or added a regression test if fixing a bug
- [ ] All tests pass
- [x] Run the backend locally and it works
- [x] Added migrations if there are any database changes, rebased onto `develop` if necessary for linear migration history

### CODE REVIEW

File: app/backend/src/couchers/migrations/versions/1c809d111871_1996_language_list_yue.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+revision = "1c809d111871"
+down_revision = "1e9b694d08c0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    session = Session(bind=op.get_bind())
+    copy_resources_to_database(session)
+    op.execute("UPDATE languages SET name = 'Chinese (Cantonese/Yue)' WHERE code = 'yue';")
```
Comment (aapeliv) [inline app/backend/src/couchers/migrations/versions/1c809d111871_1996_language_list_yue.py]: You don't need to issue this UPDATE: the `copy_resources_to_database` function replaces the `languages` table and does that already!

**Review by aapeliv — CHANGES_REQUESTED** [review]
Hi @foormea, thank you for this PR and for getting involved! I've also just replied to your email.

There is one small change, otherwise this is good to go!

File: app/backend/src/couchers/migrations/versions/1c809d111871_1996_language_list_yue.py (hunk 1 above)
Comment (ghost) [reply app/backend/src/couchers/migrations/versions/1c809d111871_1996_language_list_yue.py]: 👍

**Review by aapeliv — APPROVED** [review]
LGTM!

**Pull request merged**